### ISSUE TITLE
max_block_send_distance seems to be forced to be 1

### ISSUE THREAD

**Issue by fluxionary** [obl3pplifp/BlS-issue-tracker#225]
The configuration says 4, which has been true for at least the last year. I suspect it's something to do w/ multicraft, but I don't know how. Other multicraft servers clearly are sending mapblocks at a larger distance. 

In particular, this issue makes it impossible for players w/out flight to see large builds e.g. aleksbuilder and new spawn.

**Comment by luk3yx** [comment]
Blocky Survival is currently using Minetest and I don't think this issue has disappeared.

**Comment by fluxionary** [comment]
Yeah, I'd also noticed that, thanks for updating the tab. Back to having no theory why this is an issue.
